go/README.md

```diff
@@ -1 +1,177 @@
-# Go
\ No newline at end of file
+# Go Code Conventions
+
+## Tooling
+
+- Visual Studio Code
+- delve
+- golangci-lint
+- staticcheck
+- modd
+
+## Modules
+
+go mod init bitbucket.org/aanzeeonline/[yourprojectname]
+
+Working with a local fork of a module, put an extra line in the go.mod:
+// replace bitbucket.org/aanzeeonline/libgoaz => ../LibGOAZ
+(and remove the comments)
+
+Before pushing the final update to git, make sure that the fork isn't in use anymore
+
+## project structure for an API / Web application
```

Comment:
Start headline with capitalized letter ("Project structure for an API / Web application")

Reply:
Okay